sdk/openai/Azure.AI.OpenAI/README.md

```diff
@@ -26,21 +21,68 @@ dotnet add package Azure.AI.OpenAI --prerelease
 
 ### Prerequisites
 
-Include a section after the install command that details any requirements that must be satisfied before a developer can [authenticate](#authenticate-the-client) and test all of the snippets in the [Examples](#examples) section. For example, for Cosmos DB:
-
-> You must have an [Azure subscription](https://azure.microsoft.com/free/dotnet/) and [Cosmos DB account](https://docs.microsoft.com/azure/cosmos-db/account-overview) (SQL API). In order to take advantage of the C# 8.0 syntax, it is recommended that you compile using the [.NET Core SDK](https://dotnet.microsoft.com/download) 3.0 or higher with a [language version](https://docs.microsoft.com/dotnet/csharp/language-reference/configure-language-version#override-a-default) of `latest`.  It is also possible to compile with the .NET Core SDK 2.1.x using a language version of `preview`.
+You must have an [Azure subscription](https://azure.microsoft.com/free/dotnet/) and [OpenAI access](https://learn.microsoft.com/azure/cognitive-services/openai/overview#how-do-i-get-access-to-azure-openai).
 
 ### Authenticate the client
 
-If your library requires authentication for use, such as for Azure services, include instructions and example code needed for initializing and authenticating.
+In order to interact with the App Configuration service, you'll need to create an instance of the [OpenAIClient][openai_client_class] class. To make this possible, you'll need the endpoint string for your OpenAI resource, as well as your Azure Subscription Key.
+
+#### Get credentials
+
+You can obtain the endpoint string and subscription key from the Azure OpenAI Portal.
+
+#### Create OpenAIClient
+
+Once you have the value of the endpoint string and subscription key, you can create the OpenAIClient:
 
-For example, include details on obtaining an account key and endpoint URI, setting environment variables for each, and initializing the client object.
+```C# Snippet:CreateOpenAIClient
+// Replace with your Azure subscription key
+string key = "YOUR_AZURE_KEY";
```

Comment:
Since we're going more in depth, open question with @m-nash likely knowing best of contemporary participants: is there general guidance on when docs/sample collateral such as this should use implicit typing (`var`) versus explicit types? My developer intuition would be to use `var` for string literals and direct constructor use (all LHS in this block) and this would feel "stiff" to me to read; but if we gravitate towards "stiff" to accommodate things like that discussed `Response<T>` magic and its pitfalls, then "when in Rome."

Reply:
In an IDE setting var is fine because its easy to hover and see exactly what type the variable actually is.  In docs that's a lot harder to do so as a general rule we use explicit variable types to make it as clear as possible what is happening.

Part of this is also consistency with the other API documentation in the repo.  Although there might be outliers that slipped through we try to keep a consistent style across all libraries in this repo.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/33789

Reply:
We keep they explicit type for consistency's sake.

Reply:
Double check that we don't use `var` in any of the `*.md` files